config/locales/en.yml

```diff
@@ -774,6 +775,17 @@ en:
     site_uploads:
       delete: Delete uploaded file
       destroyed_msg: Site upload successfully deleted!
+    software_updates:
+      critical_update: Critical — please update quickly
+      description: It is recommended to keep your Mastodon installation up to date to benefit from the latest fixes and features. Moreover, it is sometimes critical to update Mastodon in a timely manner to avoid security issues. For these reasons, Mastodon checks for updates every 30 minutes, and will notify you according to your e-mail notification preferences.
```

Comment:
I don't know. This should definitely go into the documentation, but we are currently not going into any specifics and I am not sure this is the correct place. As for the requests being anonymous, this wording may be a bit too broad since we don't uses proxies to mask the IP address or anything.

Reply:
Updated with a link to https://docs.joinmastodon.org/admin/upgrading/#automated_checks (the specific anchor/section does not exist yet).

![image](https://github.com/mastodon/mastodon/assets/384364/70584b58-35e5-48c2-b5e7-2fc5cb1deb9d)